frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/status-card.tsx

```diff
@@ -47,6 +54,44 @@ export const CephAlerts = withDashboardResources(
   },
 );
 
+const CephHealthChecks: React.FC<CephHealthChecksProps> = ({ cephDetails, cephHealthState }) => {
+  const { t } = useTranslation();
+  const pattern = /[A-Z]+_*|error/g;
+  const healthChecks: CephHealthCheck[] = [];
+  for (const key in cephDetails) {
+    if (pattern.test(key)) {
+      const healthCheckObject: CephHealthCheck = {
+        details: cephDetails[key].message,
+        troubleshootLink: whitelistedHealthChecksRef[key] ?? null,
+      };
+      healthChecks.push(healthCheckObject);
+    }
+  }
+
+  return (
+    <div>
```

Comment:
```suggestion

```

Reply:
Getting an error when removing it.

```
ERROR in /home/rexagod/repositories/console/frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/status-card.tsx(77,7):
TS2322: Type '({ cephDetails, cephHealthState }: PropsWithChildren<CephHealthChecksProps>) => Element[]' is not assignable to type 'FC<CephHealthChecksProps>'.
  Type 'Element[]' is missing the following properties from type 'ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)>) | (new (props: any) => Component<any, any, any>)>': type, props, ke
```

Reply:
yes because you retirning array of elements not an react element

Reply:
Check https://github.com/openshift/console/pull/9426#pullrequestreview-714248972
It will fix that